project-fortis-services/src/resolvers/Settings/mutations.js

```diff
@@ -396,18 +396,11 @@ function modifyBlacklist(args, res) { // eslint-disable-line no-unused-vars
     const mutations = [];
     const filterRecords = [];
     termFilters.forEach(termFilter => {
-      if (termFilter.id) {
-        mutations.push({
-          query: 'UPDATE fortis.blacklist SET conjunctivefilter = ?, islocation = ? WHERE id = ?',
-          params: [termFilter.filteredTerms, termFilter.isLocation, termFilter.id]
-        });
-      } else {
-        termFilter.id = uuid();
-        mutations.push({
-          query: 'INSERT INTO fortis.blacklist (id, conjunctivefilter, islocation) VALUES (?, ?, ?)',
```

Comment:
If we remove the insert query, how will we be able to add new blacklist items?

Reply:
`UPDATE` still will add the non-existent terms because cassandra update is like an upsert: https://docs.datastax.com/en/cql/3.3/cql/cql_reference/cqlUpdate.html 
`UPDATE writes one or more column values to a row in a Cassandra table. Like INSERT, UPDATE is an upsert operation: if the specified row does not exist, the command creates it. All UPDATEs within the same partition key are applied atomically and in isolation.`

Reply:
Nice. Good to know.